go/worker/keymanager/config/config.go

```diff
@@ -7,6 +7,14 @@ type Config struct {
 	RuntimeID string `yaml:"runtime_id"`
 	// Base64-encoded public keys of unadvertised peers that may call protected methods.
 	PrivatePeerPubKeys []string `yaml:"private_peer_pub_keys"`
+
+	// Churp is configuration for Churp application.
```

Comment:
Did this leak? It shouldn't be part of this PR.

Reply:
Yes, will remove.